rbac/management/workspace/model.py

```diff
@@ -38,3 +38,14 @@ class Workspace(TenantAwareModel):
 
     class Meta:
         ordering = ["name", "modified"]
+
+    def ancestors(self):
```

Comment:
The way this logic _was_, yes, otherwise we'd potentially have the ability to create an infinite loop. With [9061c6c](https://github.com/RedHatInsights/insights-rbac/pull/1189/commits/9061c6c870d0342e77379f09b70aaed9ebe34e21) we should be ok without them. In testing we'd just simulate a root without a parent.